spec/shadow/index.html

```diff
@@ -1139,6 +1141,27 @@
           </li>
         </ol>
 
+        <p>Likewise, the <a>sequential focus navigation order</a> for <a>distributed nodes</a>
```

Comment:
Wrong... This should not be *distributed nodes*. This should be *assigned nodes*.
Also do not forget to include a descendants of assigned nodes here.